tomcat-mixin/dashboards/tomcat-overview.libsonnet

```diff
@@ -0,0 +1,767 @@
+    ),
+    prometheus.target(
+      'tomcat_threadpool_keepalivecount{job=~"$job", instance=~"$instance", protocol=~"$protocol", port=~"$port"}',
+      datasource=promDatasource,
+      legendFormat='{{job}} - {{instance}} - {{protocol}}-{{port}} - idle',
+    ),
+    prometheus.target(
+      'tomcat_threadpool_currentthreadcount{job=~"$job", instance=~"$instance", protocol=~"$protocol", port=~"$port"}',
+      datasource=promDatasource,
+      legendFormat='{{job}} - {{instance}} - {{protocol}}-{{port}} - active',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Threads',
+  description: 'The number of various threads being used by a Tomcat connector',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local logsPanel = {
+  datasource: lokiDatasource,
+  targets: [
+    {
+      datasource: lokiDatasource,
+      editorMode: 'code',
+      expr: '{filename="/var/log/tomcat9/catalina.out", job=~"$job", instance=~"$instance"} |= ``',
+      queryType: 'range',
+      refId: 'A',
+    },
+  ],
+  type: 'logs',
+  title: 'Logs',
+  description: 'Recent logs from the Catalina.out logs file\n',
+  options: {
+    dedupStrategy: 'none',
+    enableLogDetails: true,
+    prettifyLogMessage: false,
+    showCommonLabels: false,
+    showLabels: false,
+    showTime: false,
+    sortOrder: 'Descending',
+    wrapLogMessage: false,
+  },
+};
+
+{
+  grafanaDashboards+:: {
+    'overview.json':
+      dashboard.new(
+        'Overview',
+        time_from='%s' % $._config.dashboardPeriod,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        description='',
+        uid=dashboardUid,
+      )
+
```

Comment:
we can add a link to other dashboards here. take a look at another mixin like cassandra

Reply:
FYI seems like dashboard-to-mixin tool doesn't keep these links, need to be added manually to libsonnet files